packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -44,6 +44,14 @@ When you create an Okta expression, you can reference any property that exists i
 | `user.$property`                  | `user` reference to the Okta user<br>`property` the top-level property variable name<br>`id`, `status`, `created`, `lastUpdated`, `passwordChanged`, `lastLogin`   | `user.id`<br>`user.status`<br>`user.created`   |
 | `user.profile.$profile_property`  | `profile_property` reference to user profile property, including custom-defined properties  | `user.profile.firstName`<br>`user.profile.email`<br>           |
 
+### Okta Device Profile
+When you create an Okta expression, you can reference any property that exists in an Okta Device Profile, and EDR attributes.
```

Comment:
We should have a link to EDR here https://help.okta.com/oie/en-us/Content/Topics/identity-engine/devices/edr-integration-main.htm

Reply:
+1, please add link to generic EDR integration as well.

Reply:
@jane-kuang-okta the doc link was added. Please see attached image.

Reply:
@sshen-okta Just saw the updated image. The link that Santosh is referring to is the _general_ EDR integration doc; you added a different EDR _signals_ link that Apoorva's commented on. I think it would be good to add the general EDR link as well.

Reply:
Added both link @jane-kuang-okta, please see attached image.